### ISSUE TITLE
"Add widget" buttons duplicate

### ISSUE THREAD

**Issue by JakubChecinski** [LemurPwned/spintronics-visual#137]
I'm not 100% sure it's a bug we need to fix, but dr Frankowski told me to report it just in case

**Indicate the dataset**
Single files from 0200 nm dataset

**To Reproduce**
1. Load a single file
2. Load another single file
3. For the next few seconds, the "add widget" buttons duplicate
4. After a few seconds, there is only one button and it can be clicked normally

**Screenshots**
https://imgur.com/bbryCuW

**Comment by LemurPwned** [comment]
Yes, it is a known bug. However, we do not know how to fix it :( (yet!)
Marking is as wontfix for now
